files/scripts/asic_status.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+"""
+    bootstrap-asic
+"""
+try:
+    import re
+    import syslog
```

Comment:
Better to use the logger in sonic-py-common than directly calling syslog.

Reply:
Done.